Code readability update

D0048 opened the pull request:
You can just merge the .c files if you like

D0048 committed:
Update README.md

D0048 committed:
Update README.md

D0048 committed:
Code readability enhance

jaseg commented:
That can be fixed easily! (have a look at your inbox)

jaseg commented:
Content-wise, I don't think a ctags file should be part of the repo as it is basically the same as other compiler output.

D0048 commented:
In my opinion, a ctags file will largely help the developers who work across devices (like me), and probably won't affect anyone who does not need it anyways. However, I can keep it out of your branch if you really want. : )
By the way, thanks for your work! I am just new to github and trying to help.

D0048 committed:
Make file update

Modified install to all $PATH accordingly
Modified remove from all $PATH accordingly

D0048 committed:
Update custom password feature

D0048 committed:
Update manual, Update function: custom password

Now stable for use, with debug output.
